build/wix/main.wxs

```diff
@@ -157,6 +157,18 @@
       <ComponentRef Id="PathUser" />
       <ComponentRef Id="PathSystem" />
     </Feature>
+    <UI>
+       <UIRef Id="WixUI_RD" />
+       <Publish Dialog="ExitDialog"
+         Control="Finish"
+         Event="DoAction"
+         Value="LaunchApplication"
+       >WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT = 1 and NOT Installed</Publish>
+    </UI>
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Run Rancher Desktop" />
```

Comment:
```suggestion
    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Run [ProductName]" />
```
Not that we'd be renaming the product any time soon, but it seems more tidy to do it.

Reply:
This doesn't work. I tried creating a custom action with `Property` `"ProductName"` and value `"Rancher Desktop"` but that didn't work either. Is there a specific way to do this?

Reply:
The reference to `[ProductName]` in a Value attribute raises an error message. And putting `"[#ProductName]"` verbatim in that attribute results in `[#ProductName]` showing up in the checkbox text.

Reply:
Ah, I couldn't get this to work either :( I guess the only thing we could do would be to define a preprocessor variable and do `$(var.productName)` instead…